src/data/onChainPoolDataViaReadContract.ts

```diff
@@ -1,8 +1,20 @@
-import { PublicClient, parseAbi, Address, Abi, Hex } from 'viem';
+import { PublicClient, Address, Abi, Hex } from 'viem';
 import { getPoolAddress } from '../utils';
 import { OnChainPoolData } from './enrichers/onChainPoolDataEnricher';
 import { SwapOptions } from '../types';
 
+import ComposableStablePoolV5Abi from '../abi/abis/ComposableStablePoolV5Abi.json';
+import ConfigurableRightsPoolAbi from '../abi/abis/ConfigurableRightsPoolAbi.json';
+import ConvergentCurvePoolAbi from '../abi/abis/ConvergentCurvePoolAbi.json';
```

Comment:
I'm not familiar with these abis - can you give more details on where you did you get them?